runtime.md

```diff
@@ -54,17 +56,28 @@ The lifecycle describes the timeline of events that happen from when a container
 3. Once the container is created additional actions MAY be performed based on the features the runtime chooses to support.
    However, some actions might only be available based on the current state of the container (e.g. only available while it is started).
 4. Runtime's `start` command is invoked with the unique identifier of the container.
-   The runtime MUST run the user-specified code, as specified by [`process`](config.md#process-configuration).
-5. The container's process is stopped.
+5. The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
```

Comment:
So to be clear on options for resolving this, we have:

a. Keep this line as it is, with `prestart` firing after `start` was called but before the meat of the `start` action (execing the user-specified code).
b. Shift `prestart` to fire at the end of `create`.
    i. And still call it `prestart`.
    ii. And rename it to `postcreate`.

I'm ok with (a) and (b.ii).  I think (b.i) will have a more impact with users resolving confusion, while (b.ii) will have a very small impact on existing hook consumers who have to rename their config settings.